addons/mail/static/src/core/common/chat_bubble.xml

```diff
@@ -38,14 +35,4 @@
         </div>
     </t>
 
-    <!-- @type {import("models").Message} message -->
-    <t t-name="mail.message_preview_prefix">
-        <t t-if="message.isSelfAuthored">
-            <i class="fa fa-mail-reply me-1 opacity-75"/>You:
-        </t>
-        <t t-elif="!message.author?.eq(channel?.correspondent?.persona)">
-            <t t-esc="message.authorName"/>:
-        </t>
-    </t>
-
 </templates>
```

Comment:
```suggestion
<i t-if="message.hasOnlyAttachments" class="fa me-1" t-att-class="message.previewIcon"/>
<t t-out="message.previewText" />
```

Not code in diff, but close to it and could take opportunity to make code simpler as it should.